packages/lodestar/src/chain/blocks/verifyBlock.ts

```diff
@@ -290,5 +288,10 @@ export async function verifyBlockStateTransition(
     });
   }
 
-  return {postState, executionStatus};
+  const isMergeTransitionBlock =
```

Comment:
Why has this variable been moved to verifyBlock?

Reply:
need preState to evaluate it, on postState it was always coming as false as on merge block poststate will have merge complete